quiche/src/recovery/mod.rs

```diff
@@ -24,31 +24,22 @@
 // NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
 // SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-use std::cmp;
-
+use std::str::FromStr;
 use std::time::Duration;
 use std::time::Instant;
 
-use std::collections::VecDeque;
-
-use crate::packet::Epoch;
-use crate::ranges::RangeSet;
-use crate::Config;
-use crate::CongestionControlAlgorithm;
-use crate::Result;
-
 use crate::frame;
-use crate::packet;
-use crate::ranges;
+use crate::Config;
 
 #[cfg(feature = "qlog")]
 use qlog::events::EventData;
 
 use smallvec::SmallVec;
 
-use self::congestion::pacer;
-use self::congestion::Congestion;
-use self::rtt::RttStats;
+mod congestion;
```

Comment:
Nit: general convention in quiche is to put the `mod`s at the end of the file.

Reply:
Done.